draft-ietf-masque-h3-datagram.md

```diff
@@ -361,6 +367,9 @@ feature. Implementations that support this feature SHOULD always send this
 Settings parameter to avoid leaking the fact that there are applications using
 HTTP/3 datagrams enabled on this endpoint.
 
+Since use of the Capsule Protocol is restricted to new HTTP Upgrade Tokens, it is
+not accessible from JavaScript.
```

Comment:
Do we really mean all of JavaScript, or are we talking about not directly accessible rrom Web Platform APIs (Fetch, XHR etc)?

Reply:
Agreed, fixed.